static/locales/en-US.yaml

```
@@ -1017,8 +1017,7 @@ Tooltips:
     Ignore Default Arguments: Do not send any default arguments to the external player
       aside from the video URL (e.g. playback rate, playlist URL, etc.).
       Custom arguments will still be passed on.
-    Custom External Player Arguments: Any custom command line arguments, separated by semicolons (';'),
-      you want to be passed to the external player.
+    Custom External Player Arguments: Any custom command line arguments you want to be passed to the external player.
```

Sound better in my head

```suggestion
    Custom External Player Arguments: Any custom command line arguments you want to be passed on to the external player.
```

---

Done :)